admin/includes/auto_loaders/config.https.php

```diff
@@ -0,0 +1,3 @@
+<?php
+$autoLoadConfig[15][] = array('autoType' => 'init_script',
+                              'loadFile' => 'init_https.php');
```

Comment:
To share it from catalog side:
```diff
-                              'loadFile' => 'init_https.php');
+                              'loadFile'=> DIR_FS_CATALOG . 'includes/init_includes/init_https.php');
```

Reply:
not true.  as i suspected there is no way to share an init_script from the customer side to the admin side.  you must change the autoType.  from autoload_func.php:

      case 'init_script':
      $baseDir = DIR_WS_INCLUDES . 'init_includes/';
      if (file_exists(DIR_WS_INCLUDES . 'init_includes/overrides/' . $entry['loadFile'])) {
        $baseDir = DIR_WS_INCLUDES . 'init_includes/overrides/';
      }
      /**
       * include an init_script as specified by autoloader array
       */
      require($baseDir . $entry['loadFile']);
      $debugOutput .= 'require(\'' . $baseDir . $entry['loadFile'] . '\');' . '<br />';
      break;


one must change the autoType to either require or include.  is there a preference?  i'mn guessing require as that is what the init_script autoType does...